project.clj

```diff
@@ -139,7 +139,9 @@
                                        :main-namespace "puppetlabs.puppetdb.core"
                                        :create-varlib true}
                                 :config-dir "ext/config/pe"}
-                  :version ~pe-pdb-version
+                  :plugins [[puppetlabs/lein-ezbake "0.2.8"
+                              :exclusions [org.clojure/clojure]]]
```

Comment:
It does require VPN access to use the build functionality (or if a dependency needs to be pulled from the internal repo), it could be used to generate the packaging artifacts that could be used externally, with some effort on the part of the user, however you're right it might be better to leave it tied to the profiles that require it.